cli/cmd/cluster.go

```diff
@@ -61,6 +62,8 @@ var (
 	_flagClusterDownKeepVolumes bool
 )
 
+var _eksctlPrefixRegex = regexp.MustCompile("^[0-9]{4}-[0-9]{2}-[0-9]{2} [0-9]{2}:[0-9]{2}:[0-9]{2} \\[.+] {2}")
```

Comment:
I am not sure but you may be able to avoid `\\` if you use backticks instead of `"` around the regex

Reply:
the IDE escaped it for me, I'm not sure either... I'm not a regex expert, at all 😅

Reply:
`"^[0-9]{4}-[0-9]{2}-[0-9]{2} [0-9]{2}:[0-9]{2}:[0-9]{2} \\[.+] {2}"` -> 
```
`^[0-9]{4}-[0-9]{2}-[0-9]{2} [0-9]{2}:[0-9]{2}:[0-9]{2} \[.+] {2}`
```